types/koa-helmet/index.d.ts

```diff
@@ -0,0 +1,62 @@
+// Type definitions for koa-helmet 3.1
+// Project: https://github.com/venables/koa-helmet#readme
+// Definitions by: Nick Simmons <https://github.com/me>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import {
+    IHelmetConfiguration,
+    IHelmetFrameguardConfiguration,
+    IHelmetHstsConfiguration,
+    IHelmetXssFilterConfiguration,
+    IHelmetDnsPrefetchControlConfiguration,
+    IHelmetHpkpConfiguration,
+    IHelmetReferrerPolicyConfiguration } from 'helmet';
+import {Middleware, Context} from 'koa';
+
+export type KoaHelmetContentSecurityPolicyDirectiveFunction = (ctx: Context) => string;
```

Comment:
This type is why i had to pull all the rest of HelmetCsp related stuff. Needed to convert function from Express to Koa handler.